config.js

```diff
@@ -82,30 +82,31 @@ sites:{
   "domaene74" : {id: "domaene74",short:"d74",name:"Recklinghausen", version:"4.1.0", version_base:"2018.2.2", on_map: true},
   "domaene75" : {id: "domaene75",short:"d75",name:"Gronau", version:"4.1.0", version_base:"2018.2.2", on_map: true},
   "domaene76" : {id: "domaene76",short:"d76",name:"Herne", version:"4.1.0", version_base:"2018.2.2", on_map: true},
+  "domaene77" : {id: "domaene77",short:"d77",name:"Hamm", version:"4.1.0", version_base:"2018.2.2", on_map: true},
 },
 
 //router list for gluon v2018.2
 manufacturers: {
   "0tp-link": {id: "tp-link", name: "TP-Link"},
   "1ubiquiti": {id: "ubiquiti", name: "Ubiquiti"},
   "2avm": {id: "avm", name: "AVM"},
-  "2zyxel": {id: "zyxel", name: "Zyxel"},
-  "3allnet": {id: "allnet", name: "Allnet"},
-  "3buffalo": {id: "buffalo", name: "Buffalo"},
-  "4d-link": {id: "d-link", name: "D-Link"},
-  "5gl-inet": {id: "gl-inet", name: "GL-Inet"},
-  "5gl-inet": {id: "gl", name: "GL Innovations"},
-  "6linksys": {id: "linksys", name: "Linksys"},
-  "7netgear": {id: "netgear", name: "Netgear"},
-  "8alfa": {id: "alfa-network", name: "ALFA-Network"},
-  "98devices": {id: "8devices", name: "8devices"},
-  "10meraki": {id: "meraki", name: "Meraki"},
-  "10nexx": {id: "nexx", name: "Nexx"},
-  "11openmesh": {id: "openmesh", name: "OpenMesh"},
-  "12onion": {id: "onion", name: "Onion"},
-  "13wd-ny-net": {id: "wd-ny-net", name: "WD My Net"},
-  "13wd-ny-net": {id: "raspberrypi", name: "Raspberry Pi"},
-  "14x86":{id:"x86",name:"x86"}
+  "3zyxel": {id: "zyxel", name: "Zyxel"},
+  "4allnet": {id: "allnet", name: "Allnet"},
+  "5buffalo": {id: "buffalo", name: "Buffalo"},
+  "6d-link": {id: "d-link", name: "D-Link"},
+  "7gl-inet": {id: "gl-inet", name: "GL-Inet"},
+  "8gl-innovations": {id: "gl", name: "GL Innovations"},
+  "9linksys": {id: "linksys", name: "Linksys"},
+  "10netgear": {id: "netgear", name: "Netgear"},
+  "11alfa": {id: "alfa-network", name: "ALFA-Network"},
```

Comment:
Wollen wir nicht einfach nur auf Zahlen umstellen? Diese Nummerierung+Name ist irgendwie komisch.

Reply:
Dafür. Vermutlich wurde das gemacht weil es so einfacher ist eine Zeile zwischen zu fügen ohne die nachfolgenden aufrutschen zu müssen.

Reply:
Nur wenn man nummeriert, dann sollte man es auch durchhalten, daher der PR.

Ich finde die Nummerierung grundsätzlich nicht schön, weil sie keinen Mehrwert liefert.
Wenn man neue Werte einfügen möchte der die Reihenfolge der Einträge verändern will, dann ändern muss man, um die aufsteigende Nummerierung durchzuhalten immer x Zeilen aktualisieren. Das erzeugt im Review nur unnötiges Rauschen.

Eigentlich ist es ja nur wichtig das jeder Key eindeutig ist - hierzu würde ich gerne schlicht den **Wert aus "id" für den key wiederverwenden**/spiegeln. Das würde ich machen, damit der Testaufwand gering ist.

Wenn mal einer Lust das JavaScript-Drumherum anzupassen, dann könnte man das Object aus zu einem Array of Object machen (ohne keys; nur noch Elemente, die aus id+name Objekten bestehen). Das mache ich aber nicht.

Viel wichtiger finde ich aber die **User Experience** und da wäre es schön, wenn man **gemäß "name" alphabetisch sortiert** - entweder im Script oder **direkt hier in der config**. Letztes würde ich machen - damit der Test-Aufwand gering ist.

Was meint Ihr?

Reply:
Es gibt quasi nur zwei Marken, nah gut inzwischen drei, die verwendet werden: TP-Link, Ubiquiti und AVM. Die sollen oben stehen. Daher haben wir es nicht alphabetisch sortiert sondern diese komischen Zahlen davor gesetzt.

Reply:
Ah, ok, wenn es um die Sortierung geht, dann wäre es gut, wenn das JavaScript kein "sort by key"  macht und einfach die Reihenfolge nimmt, wie sie in der Config vorkommt.

Wenn man das JavaScript aber nicht anpassen möchte (ich finde die Code Stelle gerade nicht), dann wäre ich dafür das zumindest im Keynamen (semantisch) kenntlich zu machen - z.B. `position01`, `position02`, etc. bzw. mehr im Streuspeicherverfahren:  `position0100`, `position0200`, damit man nachträglich Einträge (mittig) ergänzen kann, z.B. `position0150`, noch später dann `position0125` oder `pos0175`, um nicht bei jeder Einfügung gleich "alle" anderen keys neu durchnummerieren zu müssen.

Reply:
führende Nullen würden ansonsten ggf. abgeschnitten